src/registry/licensePlate/index.stories.tsx

```diff
@@ -0,0 +1,210 @@
+import {LicensePlateComponentSchema} from '@open-formulieren/types';
+import type {Meta, StoryObj} from '@storybook/react';
+import {expect, fn, userEvent, within} from '@storybook/test';
+
+import {FormioFormProps} from '@/components/FormioForm';
+import {renderComponentInForm} from '@/registry/storybook-helpers';
+import {withFormik} from '@/sb-decorators';
+
+import {FormioLicensePlate as LicensePlate} from './';
+import ValueDisplay from './ValueDisplay';
+
+export default {
+  title: 'Component registry / custom / licensePlate',
```

Comment:
```suggestion
  title: 'Component registry / custom / licenseplate',
```
(for parity with the `type` key value)